commit 3537454e66c3d9466d9d574467573e4ba4333293
Author: Cameron Nemo <camerontnorman@gmail.com>
Date:   Sun Oct 28 01:59:05 2018 -0700

    nih/signal.c: only support SIGCLD if it has libc support

diff --git nih/signal.c nih/signal.c
index a241df9..6a263fa 100644
--- a/nih/signal.c
+++ b/nih/signal.c
@@ -87,7 +87,9 @@ static const SignalName signal_names[] = {
 	{ SIGSTKFLT, "STKFLT" },
 #endif
 	{ SIGCHLD,   "CHLD"   },
+#ifdef SIGCLD
 	{ SIGCLD,    "CLD"    },
+#endif
 	{ SIGCONT,   "CONT"   },
 	{ SIGSTOP,   "STOP"   },
 	{ SIGTSTP,   "TSTP"   },
